src/components/ListRepositories.tsx

```diff
@@ -4,24 +4,24 @@ import Skeleton from "react-loading-skeleton";
 
 export declare interface ListRepositoriesProps {
   activeLink: string | null;
-  limit: number;
   handleLoadingMore: () => void;
   fetchedData: DbRepo[];
   title: string;
+  hasNextPage: boolean;
 }
 
 const ListRepositories = ({
   activeLink,
-  limit,
   handleLoadingMore,
   fetchedData,
   title,
+  hasNextPage,
 }: ListRepositoriesProps): JSX.Element => {
   if (!fetchedData.length) {
     return (
       <div className="mx-auto max-w-7xl px-4 mt-10">
         <div className="flex flex-col gap-y-5 overflow-hidden mb-12">
-          {Array.from(Array(10).keys()).map(item => (
+          {Array.from(Array(25).keys()).map(item => (
```

Comment:
it seems that in the past I see that it was 10 why did you change it to 25?? does this have a feature ??

Reply:
I made changes to match the length of the fetchedData list, enhancing the user experience. With the array length set to 25, users no longer need to scroll down to see the load more button.

I have included a video demonstrating the current behaviour. I hope this will provide clarity.
[Screencast from 20-07-23 11:15:43 AM IST.webm](https://github.com/open-sauced/hot/assets/25549935/8c3fc707-64bc-4e5e-a79a-6c408b1fdea9)

Reply:
@bdougie are you ok with this change?